customer-satisfaction/README.md

```diff
@@ -39,35 +43,45 @@ Our standard training pipeline consists of several steps:
 - `model_train`: This step will train the model and save the model using [MLflow autologging](https://www.mlflow.org/docs/latest/tracking.html).
 - `evaluation`: This step will evaluate the model and save the metrics -- using MLflow autologging -- into the artifact store.
 
-### Deployment & Inference Pipeline
+### Deployment Pipeline
+
+We have another pipeline, the `deployment_pipeline.py`, that extends the training pipeine, and implements a continuous deployment workflow. It ingests and processes input data, trains a model and then (re)deploys the prediction server that serves the model if it meets our evaluation criteria. For us this is the [MSE](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html); you can also set your own minimum MSE. The first four steps of the pipeline are the same as above, but we have added the following additional ones:
```

Comment:
Here Pipeline spelling is wrong, you have written `pipeine` but it should be pipeline.